### DISCUSSION TITLE
ArozOS v2.00x Updates - Breaking Changes Worth Noticing

### DISCUSSION THREAD

**Discussion by tobychui** [tobychui/arozos#101] [General]
![圖片](https://user-images.githubusercontent.com/24617523/185778762-635cf459-4ee1-4768-9b02-623cda27bb3b.png)
## ArozOS v2.00x Updates - Breaking Changes Worth Noticing 

Hi everyone,

This is a discussion opened specifically for ArozOS v2.00x. 
In this new version, we will have (lots) of breaking changes related to storage pool architectures so we can embed new file system abstractions into ArozOS, including mounting WebDAV / SMB network storage as local storage roots, dedicated share managers etc. As a side effect, some functions like Hybrid-backup and AGI's virtual path to real path decoder will be deprecated.

**As we do not have any information collection function in ArozOS that tracks who is using these functions, we (developers) will need your input on what changes might breaks most of your things** and we will try our best to make the transition as smooth as possible with a few more updates to the remaining v1.12x release schedule.

To sum up, this will be a great update to ArozOS and it might make this project the world first, cross platform, cross cloud provider distributed web desktop system. Feel free to raise your idea, concern or insights on the new system. We are looking forward to discuss any new design changes with you 👍🏻 

Wish all of you the best, 
Toby Chui
Primary Author of ArozOS Project

**Comment by pi-dal** [comment]
You have done a very creative job, good luck! :thumbsup:
